### PR TITLE
Added method `createFxpRateForCurrencyChannel`;

### PR DESCRIPTION
**Pull request by vbarzokas**
Added method `createFxpRateForCurrencyChannel`;
Fixed URL for `getFxpRatesForChannel`;
Bumped to 0.0.4.

### CODE REVIEW

File: src/admin/api.js [hunk 1]
```diff
@@ -305,7 +338,28 @@ function buildCurrencyChannelRates(rates) {
     });
 }
 
+/**
+ * @method extractSourceCurrency
+ * @private
+ * @param {string} currencyPair
+ * @return {string}
+ */
+function extractSourceCurrency(currencyPair) {
```
[the remark this answers is not in the document]
Comment (KamuelaFranco) [reply src/admin/api.js]: “eurusd” it’s always a 6-char string, so he plucks the source and destination.

File: src/admin/api.js [hunk 2]
```diff
@@ -305,7 +338,28 @@ function buildCurrencyChannelRates(rates) {
     });
 }
 
+/**
+ * @method extractSourceCurrency
```
Comment (KamuelaFranco) [inline src/admin/api.js]: This is actually just a function, are we able to identify it that way?

**Comment by vbarzokas** [comment]
Yes I just searched for it and there seems to be a `@function` directive, I
thought there isn't. Will change it to that.

File: src/admin/api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vbarzokas) [reply src/admin/api.js]: Yes it makes sense to document that expected string format. Will do it.

File: src/admin/api.js (hunk 2 above)
Comment (vbarzokas) [reply src/admin/api.js]: Yes I just searched for it and there seems to be a `@function` directive, I thought there isn't. Will change it to that.

File: src/admin/api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (msk-) [reply src/admin/api.js]: :+1:

**Pull request merged**